Doesn't build on docs.rs anymore

https://docs.rs/crate/deno_core/0.197.0/builds/870081

This is very strange:

```
[INFO] [stderr]  Documenting deno_core v0.197.0 (/opt/rustwide/workdir)
[INFO] [stderr] error[E0463]: can't find crate for `deno_ops`
```

Maybe it's a release timing issue? The crate is definitely pushed now, but I am curious if there was a period where it wasn't available.

--- a/ops/Cargo.toml
+++ b/ops/Cargo.toml
@@ -28,7 +28,6 @@ strum_macros.workspace = true
 syn.workspace = true
 syn2.workspace = true
 thiserror.workspace = true
-v8.workspace = true
 
 [dev-dependencies]
 pretty_assertions.workspace = true

--- a/ops/op2/dispatch_slow.rs
+++ b/ops/op2/dispatch_slow.rs
@@ -734,10 +734,7 @@ fn throw_type_error(
   message: String,
 ) -> Result<TokenStream, V8MappingError> {
   // Sanity check ASCII and a valid/reasonable message size
-  debug_assert!(
-    message.is_ascii()
-      && message.len() < 1024
-  );
+  debug_assert!(message.is_ascii() && message.len() < 1024);
 
   let maybe_scope = if generator_state.needs_scope {
     quote!()

--- a/ops/op2/dispatch_slow.rs
+++ b/ops/op2/dispatch_slow.rs
@@ -736,7 +736,6 @@ fn throw_type_error(
   // Sanity check ASCII and a valid/reasonable message size
   debug_assert!(
     message.is_ascii()
-      && message.len() < v8::String::max_length()
       && message.len() < 1024
   );
 

